lib/router/mbaas.js

```diff
@@ -64,55 +64,77 @@ function initRouter(mediator, authResponseExclusionList, expressSessionMiddlewar
   });
 
   router.route('/').get(function(req, res) {
-    mediator.once('done:wfm:user:list', function(data) {
-      res.json(data);
+    mediator.request('wfm:user:list').then(function(result) {
+      if (result instanceof Error) {
+        mediator.publish('error:wfm:user:list', result);
+        return res.status(500).json({message: "Unexpected error when getting the list of users. " + result.toString()});
+      }
```

Comment:
Also it would be nice to throw an error to delegate to express' global error handler. I think the cloud apps have one setup and it's what we should be recommending